docs/javascript/v2/how-to-guides/enable-content-sharing/pdf-annotations.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Share and Annotate PDFs
+nav: 1.35
+---
+
+You can now enable PDF annotations and sharing, allowing users to share and annotate PDFs in real-time, providing a rich immersive experience without tab switching.
```

Comment:
would be good to call out that a shared PDF with annotations will be visible on all platforms (as a screen share)